bin/i18n/convert.js

```diff
@@ -0,0 +1,74 @@
+const { i18nextToPo, gettextToI18next } = require('i18next-conv');
+const fs = require('fs-extra');
+const path = require('path');
+
+/**
+ * Convert all translation files from json to po or vice versa
+ *
+ * @param {string} source - source files (json or po)
+ * @param {string} target - target files (po or json)
+ * @param {string} [base=public/locales] - directory with locale directories with translation files
+ */
+async function convert(source, target, base = 'public/locales') {
+  // find all existing translation files in source format
+  const locales = await fs.readdir(base);
+  for (const locale of locales) {
+    const files = (await fs.readdir(path.join(base, locale))).filter(
+      file => path.parse(file).ext === `.${source}`,
+    );
+
+    // transform each of the found translation files to target format
+    for (const file of files) {
+      const transformed = await (source === 'json'
+        ? i18nextToPo
+        : gettextToI18next)(
+        locale,
+        await fs.readFile(path.join(base, locale, file)),
+      );
+      await fs.writeFile(
+        path.join(base, locale, path.parse(file).name + `.${target}`),
+        transformed,
+      );
+    }
+  }
+}
+
+async function convertFile(file) {
+  const extension = path.parse(file).ext;
+
+  const extensions = ['.json', '.po'];
+
+  if (!extensions.includes(extension))
```

Comment:
Hm, odd, I expected Prettier to add `{}` here.

Reply:
Yes, interesting. Anyways, added it manually in 04dd8f8cf4264c91b06ad72a7700f35ff46309cc